src/dtx/dtx_common.c

```diff
@@ -1743,9 +1747,15 @@ stop_dtx_reindex_ult(struct ds_cont_child *cont)
 	if (dtx_cont_opened(cont))
```

Comment:
I looked dtx_cont_close() closer, looks the a "dtx_cont_open()" could be called before "stop_dtx_reindex_ult()", then we need to check dtx_cont_openend() here.

Maybe it's better to move this check out of stop_dtx_reintdex_ult(). (like checking dtx_cont_opened() before calling stop_dtx_reindex_ult() in dtx_cont_close()), and put an assert here, since the stop_dtx_reindex_ult() is called in cont_child_stop() as well.

Reply:
stop_dtx_reindex_ult() may be called after DTX resync completed globally, then there may be race with container open. It may be not impossible to adjust related logic to reduce race, but checking here is more safe and simple.